CHANGES.rst

```diff
@@ -444,6 +444,7 @@ API Changes
 
 Bug Fixes
 ^^^^^^^^^
+- `astropy.io.fits.header.Header` will no longer print an ampersand on the final CONTINUE card as per the spec [#3282].
```

Comment:
Thanks also for remembering to include a changelog entry!  Unfortunately I have to nitpick this as well:

1) We try to organize the changelog according to submodule.  As you can see, below there is a subsection for all `astropy.io.fits` fixes.  So you would add this entry down there instead, with the proper indentation (also try to break up the line so it isn't too long, but I always fix that up before release anyways so I try not to pick on people for that most of the time).

2) I would also reword it a bit.  It doesn't have to mention the `Header` class directly.  Change this to just:
```
 - CONTINUE cards no longer end the value of the final card in the series with an ampersand, per the
     specification of the CONTINUE card convention. [#3282]
```
(note, the issue # reference goes outside the period).